test/spec/Feature/Query/RelatedQueriesSpec.hs

```diff
@@ -275,3 +276,37 @@ spec = describe "related queries" $ do
         { matchStatus  = 200
         , matchHeaders = [matchContentTypeJson]
         }
+
+    it "works with count=exact" $ do
+      request methodGet "/projects?select=name,clients(name)&clients=not.is.null"
+        [("Prefer", "count=exact")] ""
+       `shouldRespondWith`
+        [json|[
+          {"name":"Windows 7", "clients":{"name":"Microsoft"}},
+          {"name":"Windows 10", "clients":{"name":"Microsoft"}},
+          {"name":"IOS", "clients":{"name":"Apple"}},
+          {"name":"OSX", "clients":{"name":"Apple"}}
+        ]|]
+        { matchStatus  = 200
+        , matchHeaders = [ matchContentTypeJson
+                         , "Content-Range" <:> "0-3/4" ]
+        }
+      request methodGet "/projects?select=name,clients()&clients=is.null"
+        [("Prefer", "count=exact")] ""
```

Comment:
Is `count=planned` and `count=estimated` relevant here? Maybe we could add a test to ensure they work too?

Reply:
Yes, those didn't work either.